app/src/main/java/org/kiwix/kiwixmobile/nav/destination/reader/ReaderFragment.kt

```diff
@@ -74,13 +78,38 @@ class ReaderFragment : CoreReaderFragment() {
     }
   }
 
+  override fun onCreateView(
+    inflater: LayoutInflater,
+    container: ViewGroup?,
+    savedInstanceState: Bundle?
+  ): View? {
+    val view = super.onCreateView(inflater, container, savedInstanceState)
+    setFragmentContainerBottomMarginToSizeOfNavBar()
+    return view
+  }
+
+  private fun setFragmentContainerBottomMarginToSizeOfNavBar() {
+    val typedValueMargin = TypedValue()
+    context?.theme?.resolveAttribute(android.R.attr.actionBarSize, typedValueMargin, true)
```

Comment:
I think you might be able to rename/reuse our extension function `Context.getColorAttribute`

Reply:
I'm in love with our extension functions. I should start looking for extension functions when encountering boilerplate!